string request
byte isRecognitionEnabled
byte isTrackingEnabled
---
string[] names
byte recognized
